client/components/place/place-list/place-list.component.js

```diff
@@ -54,28 +54,28 @@ angular.module('placeList', ['filterMapType'])
         places = result;
         counter = 1;
         for (i = 0; i < places.length; i++) {
```

Comment:
* You can use foreach here 
* it would be great to have inner json formatted like:
```javascript
{
 id: places[i]._id,
 latitude: places[i].location.coordinates[0],
 longitude: places[i].location.coordinates[1]
...
}
```